### PR TITLE
JSON import/export of Cnec result extension

### PR DESCRIPTION
**Pull request by baptsegu**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: commons/pom.xml [hunk 1]
```diff
@@ -52,6 +52,10 @@
             <groupId>com.powsybl</groupId>
             <artifactId>powsybl-tools</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.auto.service</groupId>
+            <artifactId>auto-service</artifactId>
+        </dependency>
```
Comment (terriervik) [inline commons/pom.xml]: Please respect the alphabet...

File: data/crac-io/crac-io-json/src/main/java/com/farao_community/farao/data/crac_io_json/JsonImport.java [hunk 2]
```diff
@@ -10,6 +10,7 @@
 import com.farao_community.farao.data.crac_api.Crac;
 import com.farao_community.farao.data.crac_impl.SimpleCrac;
 import com.farao_community.farao.data.crac_io_api.CracImporter;
+import com.farao_community.farao.data.crac_io_json.deserializers.SimpleCracDeserializer;
```
Comment (terriervik) [inline data/crac-io/crac-io-json/src/main/java/com/farao_community/farao/data/crac_io_json/JsonImport.java]: Is this import used?

File: flowbased-computation/flowbased-computation-impl/pom.xml [hunk 3]
```diff
@@ -37,6 +37,11 @@
         </dependency>
 
         <!-- Test dependencies -->
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-config-test</artifactId>
+            <scope>test</scope>
+        </dependency>
```
Comment (terriervik) [inline flowbased-computation/flowbased-computation-impl/pom.xml]: And the alphabet?

File: commons/pom.xml (hunk 1 above)
Comment (baptsegu) [reply commons/pom.xml]: ok

File: flowbased-computation/flowbased-computation-impl/pom.xml (hunk 3 above)
Comment (baptsegu) [reply flowbased-computation/flowbased-computation-impl/pom.xml]: ok

**Pull request merged**